games-sports/speed-dreams/patches/speed_dreams-2.2.2.patchset

```diff
@@ -0,0 +1,209 @@
+From dedc392971d6ea84a11c2f43397ecc41392ca8d8 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 18 May 2019 13:58:11 +0200
+Subject: Haiku fixes
+
+
+diff --git a/cmake/FindPLIB.cmake b/cmake/FindPLIB.cmake
+index 63b43da..9c22dbe 100644
+--- a/cmake/FindPLIB.cmake
++++ b/cmake/FindPLIB.cmake
+@@ -16,14 +16,15 @@ IF(NOT APPLE OR OPTION_USE_MACPORTS)
+  FIND_PATH(PLIB_PLIBINCLUDE_DIR plib/ssg.h
+   HINTS ENV PLIB_DIR
+   PATH_SUFFIXES 
+-	Headers include/plib include
++	Headers include/plib include develop/headers develop/headers/plib
+   PATHS
+ #Additional MacOS Paths
+-   	 ~/Library/Frameworks/plib.framework
+-  	 /Library/Frameworks/plib.framework
+- 	 /System/Library/Frameworks/plib.framework # Tiger
++	 ~/Library/Frameworks/plib.framework
```

Comment:
you could probably avoid to change these lines.

Reply:
Haven't check yet, but thought so too (looking at the file), leftover from previous patch (as the next comment) :)